core-blocks/image/index.js

```diff
@@ -217,6 +217,16 @@ export const settings = {
 			/>
 		);
 
+		if ( 'left' === align || 'right' === align ) {
+			return (
+				<aside>
+					<figure className={ classes }>
```

Comment:
To avoid code duplication, we can assign `figure` to its own variable, similar to what is done for `img` above.